test-requirements.txt

```diff
@@ -1,33 +1,34 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
```

Comment:
> By this formatting change, I think you used a newer version of the pip-tools package (pip-compile). (Maybe because pip-tools 6.2.0 with python 3.8 had some error.)

You're right.

> Anyway, I think in this case you should hand-edit pip-tools== to match the version in your environment.

For now, I've updated the file with the executed command.

I want to update `test-requirements.in`. I'm considering doing this in a separate PR.